### PR TITLE
Hides conversations when inactive

### PR DESCRIPTION
**Pull request by voloshink**

### CODE REVIEW

**Comment by Izlsnizzt** [comment]
Don't like this.

File: src/js/features/conversations.js [hunk 1]
```diff
@@ -17,6 +17,18 @@ function Conversations(timeout) {
 
     var $conversations = $('.conversations-content');
 
+    if (bttv.settings.get('hideConversations')) {
+        $conversations.css('opacity', 0);
+
+        $conversations.hover(function() {
+            $conversations.fadeTo(1, 1);
```
Comment (night) [inline src/js/features/conversations.js]: `duration` is 1ms in all your `fadeTo`s.. Explain?

**Comment by voloshink** [comment]
oh right, i was testing fades to see if it looked better, forgot to change it back to just .css

File: src/js/features/conversations.js [hunk 2]
```diff
@@ -17,6 +17,18 @@ function Conversations(timeout) {
 
     var $conversations = $('.conversations-content');
 
+    if (bttv.settings.get('hideConversations')) {
+        $conversations.css('opacity', 0);
```
Comment (night) [inline src/js/features/conversations.js]: Why opacity and not `.hide` or `.show`

**Comment by voloshink** [comment]
I think the mouseover wasn't working for me with hide/show

**Comment by night** [comment]
Have you thought to just animate it sliding down (and leaving a 1px edge to detect) and back up to reshow it?

**Comment by voloshink** [comment]
I wanted to do it, but wasn't sure if it would look good. i'll try it tonight and see if it looks fine.

**Comment by night** [comment]
merge changes making it slide and I'll take a look at the animation bug

**Pull request merged**